Scheduler/Task.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Scheduler
+{
+	/// <summary>
+	/// Task Table Info
+	/// </summary>
+	public class Task
+	{
+		public long TaskId { get; set; }
+
+		[ForeignKey("Goal")]
```

Comment:
First: this is Ok.
Second: can you implement the same logic without attributes and just using standart ef core conventions? If you can so, I think it will be nice